code/bngblaster/src/bbl_config.c

```diff
@@ -143,7 +143,7 @@ json_parse_access_line_profile(json_t *config, bbl_access_line_profile_s *profil
     json_t *value = NULL;
 
     const char *schema[] = {
-        "access-line-profile-id", "act-up", "act-down",
+        "access-line-profile-id", "draft-lihawi", "act-up", "act-down",
```

Comment:
Could you please rename `draft-lihawi` to `pon-access-line-version`, and instead of number, input should be string: `DRAFT-LIHAWI-00` or `DRAFT-LIHAWI-04`. Later we will add `RFCXXX` as soon as granted. Default remains `DRAFT-LIHAWI-04`

Reply:
You should rename variable/attribute `draft_lihawi` to `pon_access_line_version` and create an enum for the different versions. The attribute itself could be uint8_t, which is enough to hold the versions defined in the enum. 

```
typedef enum {
    DRAFT_LIHAWI_00 = 0,
    DRAFT_LIHAWI_04 = 4,
} __attribute__ ((__packed__)) pon_access_line_version_t;
```